charts/retool/templates/deployment_backend.yaml

```diff
@@ -241,7 +241,7 @@ spec:
         {{- end }}
         ports:
         - containerPort: {{ .Values.service.internalPort }}
-          name: {{ template "retool.name" . }}
+          name: http-server
           protocol: TCP
 {{- if .Values.livenessProbe.enabled }}
         livenessProbe:
```

Comment:
there is no service defined here - attempted to add, but wasn't able to get it to compile template w/out error 👀